docs/new-project.md

```diff
@@ -34,4 +34,12 @@ At this point, our project is initialized, and we're ready to start writing code
 $ dep ensure -add github.com/foo/bar github.com/baz/quux
 ```
 
-Now you're ready to move on to [Daily Dep](daily-dep.md)!
\ No newline at end of file
+### One Final Note
+
+When you've finished adding / tweaking your dependencies you should make sure to check in both the [Gopkg.toml](Gopkg.toml.md) and the [Gopkg.lock](Gopkg.lock.md).
+
+This two files work together to ensure that `dep` can provide reproducible builds. We hear you saying, "well, what about checking in vendor/".
```

Comment:
s/This/These/

Also, I think it could use some rephrasing.

Reply:
@ibrasho - no problem on the rephrasing. Which part of this would you like to see clarified?